image-layout.md

```diff
@@ -38,7 +38,7 @@ $ shasum -a 256 ./blobs/sha256-afff3924849e458c5ef237db5f89539274d5e609db5db935e
 afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256-afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51
 ```
 
-Object names in the `refs` and `blobs` subdirectories MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen `-`, the dot `.`, and the underscore `_`.
+Object names in the `refs` and `blobs` subdirectories MUST NOT include characters outside of the set of "A" to "F", "a" to "f", "0" to "9", the hyphen `-`, the dot `.`, and the underscore `_`.
```

Comment:
then perhaps we ought to have two separate sentences. Because the `blobs/` files will have a _suffix_ that is hexidecimal.
But for the `refs/` they can be the full letter set.